src/components/Map.vue

```diff
@@ -1,34 +1,36 @@
 <template>
-  <div id='turfMap' style="height: 400px"></div>
+  <div id='turfMap' style="height: 325px"></div>
```

Comment:
@rowanwins Made the map smaller to better fit the `@example` at the very bottom.